models/migrations/v68.go

```diff
@@ -56,14 +58,20 @@ func reformatAndRemoveIncorrectTopics(x *xorm.Engine) (err error) {
 			}
 
 			if models.ValidateTopic(topic.Name) {
-				log.Info("Updating topic: id = %v, name = %v", topic.ID, topic.Name)
-				if _, err := sess.Table("topic").ID(topic.ID).
-					Update(&Topic{Name: topic.Name}); err != nil {
+				count, err := sess.Where("name = ?", topic.Name).Count(&Topic{})
```

Comment:
If topic ID changes than repositories containing it should be updated from old topic ID to new one

Reply:
Added